dataeng/resources/load-vertica-schema-to-snowflake.sh

```diff
@@ -3,7 +3,7 @@
 env
 
 ${WORKSPACE}/analytics-configuration/automation/run-automated-task.sh \
- VerticaSchemaToSnowflakeTask --local-scheduler \
+ LoadVerticaSchemaFromS3ToSnowflakeTask --local-scheduler \
  --date $(date +%Y-%m-%d -d "$RUN_DATE") \
  ${OVERWRITE} \
  --credentials $SNOWFLAKE_CREDENTIALS \
```

Comment:
Note to self (@doctoryes): This file will require the addition of `scratch_schema` and `run_id`.

Reply:
Yup.    And the code as well, to pass args from Schema to Table task.